src/kernels/gpu_reference_kernel/naive_conv.cpp

```diff
@@ -83,6 +85,15 @@ inline __device__ __host__ ushort convert_fp32_to_bf16(float src_val)
     return target_val.ushortvec[1];
 }
 
+
+
+using StrideIndexType = int;
```

Comment:
Consider to make it `uint64_t` to be able to process huge tensors. Or at least unsigned.

Reply:
Jehandad and I discussed this and decided to keep 32-bit integers for now.